packages/tradershub/src/components/OptionsAndMultipliersSection/OptionsAndMultipliersContent/OptionsAndMultipliersContent.tsx

```diff
@@ -1,14 +1,24 @@
 import React, { FC } from 'react';
 import { useHistory } from 'react-router-dom';
-import { useActiveTradingAccount } from '@deriv/api';
+import { useActiveTradingAccount, useIsEuRegion } from '@deriv/api';
 import { Button, Text, useBreakpoint } from '@deriv/quill-design';
 import { optionsAndMultipliersContent } from '../../../constants/constants';
 import { getStaticUrl, getUrlBinaryBot, getUrlSmartTrader } from '../../../helpers/urls';
+import { TradingAppCardLoader } from '../../Loaders';
 import { TradingAccountCard } from '../../TradingAccountCard';
 
-type TShowButtonProps = Pick<typeof optionsAndMultipliersContent[number], 'isExternal' | 'redirect'>;
+type OptionsAndMultipliersContentItem = {
+    description: string;
+    icon: JSX.Element;
+    isExternal?: boolean;
+    redirect: string;
+    smallIcon: JSX.Element;
+    title: string;
+};
+
+type TShowButtonProps = Pick<OptionsAndMultipliersContentItem, 'isExternal' | 'redirect'>;
 
-type TLinkTitleProps = Pick<typeof optionsAndMultipliersContent[number], 'icon' | 'title'>;
+type TLinkTitleProps = Pick<OptionsAndMultipliersContentItem, 'icon' | 'title'>;
 
 const LinkTitle: FC<TLinkTitleProps> = ({ icon, title }) => {
```

Comment:
Let's not use `FC` there's no real reason why we use it Typescript already knows its a functional component
```suggestion
const LinkTitle = ({ icon, title }: TLinkTitleProps) => {
```